fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -584,7 +584,7 @@ public LogicalPlan visitFromClause(FromClauseContext ctx) {
                 LogicalPlan right = plan(relation.relationPrimary());
                 left = left == null
```

Comment:
Add parentheses to improve readability.

```suggestion
                left = (left == null)
```